specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2023-08-01-preview/appconfiguration.json

```diff
@@ -0,0 +1,2890 @@
+    },
+    "DeletedConfigurationStoreProperties": {
+      "type": "object",
+      "properties": {
+        "configurationStoreId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource id of the original configuration store."
+        },
+        "location": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The location of the original configuration store."
+        },
+        "deletionDate": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The deleted date."
+        },
+        "scheduledPurgeDate": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The scheduled purged date."
+        },
+        "tags": {
+          "readOnly": true,
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Tags of the original configuration store."
+        },
+        "purgeProtectionEnabled": {
+          "readOnly": true,
+          "type": "boolean",
+          "description": "Purge protection status of the original configuration store."
+        }
+      },
+      "description": "Properties of the deleted configuration store."
+    },
+    "DeletedConfigurationStoreListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DeletedConfigurationStore"
+          },
+          "description": "The list of deleted configuration store."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of deleted configuration stores."
+        }
+      },
+      "description": "List of deleted configuration stores"
+    },
+    "ReplicaListResult": {
+      "description": "The result of a request to list replicas.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The collection value.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Replica"
+          }
+        },
+        "nextLink": {
+          "description": "The URI that can be used to request the next set of paged results.",
+          "type": "string"
+        }
+      }
+    },
+    "Replica": {
+      "description": "The replica resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The name of the replica.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The type of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "location": {
```

Comment:
Proxy resource doesn't have location. Since the signature matches the tracked resource of ARM here, this resource is no longer being treated as Proxy. Hence, you needed to add suppressions.